forester/scripts/v2_stats.py

```
@@ -0,0 +1,214 @@
+    
+    def print_summary_stats(self) -> None:
+        """Print high-level summary statistics."""
+        print("\n" + "="*80)
+        print("V2 FORESTER TPS ANALYSIS REPORT")
+        print("="*80)
+        
+        if not self.operation_summaries:
+            print("No V2 TPS metrics found in logs")
+            return
+        
+        print(f"\nSUMMARY:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)